tests/unit/test_handlers.py

```diff
@@ -562,3 +562,34 @@ def test_s3_special_case_is_before_other_retry(self):
         self.assertTrue(s3_200_handler < general_retry_handler,
                         "S3 200 error handler was supposed to be before "
                         "the general retry handler, but it was not.")
+
+
+def _test_disable_signing_is_triggered(event):
+    responses = botocore.session.Session().emit(event)
+    if not any(handler==handlers.disable_signing for handler, _ in responses):
+        raise AssertionError("Need disable_signing() for %s" % event)
+
+def test_disable_signing_for_public_apis():
```

Comment:
Let's remove this.  These don't really add any value IMO.  This is really testing he event emitter, which we already have tests for.